_includes/foot.html

```diff
@@ -9,17 +9,17 @@ <h5 class="white-text">Come, chat with us!</h5>
       <div class="col l4 offset-l2 s12">
         <h5 class="white-text">Links</h5>
         <ul>
-          <li><a class="grey-text text-lighten-3" href="https://www.facebook.com/uqcomputing" target="_blank"><img height="14" width="14" src="{{site.baseurl}}/img/facebk.svg"></img> Facebook</a></li>
-          <li><a class="grey-text text-lighten-3" href="https://github.com/UQComputingSociety/" target="_blank"><img height="14" width="14" src="{{site.baseurl}}/img/github.svg"></img> GitHub</a></li>
-          <li><a class="grey-text text-lighten-3" href="http://twitter.com/UQComputing" target="_blank"><img height="14" width="14" src="{{site.baseurl}}/img/twtr.svg"></img> Twitter</a></li>
-          <li><a class="grey-text text-lighten-3" href="https://slack.uqcs.org.au/" target="_blank"><img height="14" width="14" src="{{site.baseurl}}/img/slack.svg"></img> Slack</a></li>
+          <li><a class="grey-text text-lighten-3" href="https://www.facebook.com/uqcomputing" target="_blank"><img height="14" width="14" src="{{site.baseurl}}/img/social/facebk.svg"></img> Facebook</a></li>
```

Comment:
Minor but I'm not a fan of these abbreviations `facbk` and `twtr`. Because if you choose to reuse it somewhere else you probably can't just remember the basic path and then replace with facebook/twitter/etc but instead would have to check the filename.

Reply:
Agreed. I have changed these to the full names :)